.travis.yml

```diff
@@ -18,44 +18,51 @@ services:
 
 cache:
   directories:
-    - sysconfcpus
-    - aion/deps
-    - aion/web/elm/elm-stuff/build-artifacts
+  - sysconfcpus
+  - aion/deps
+  - aion/web/elm/elm-stuff/build-artifacts
 
 install:
-  - cd ..
-  - |
-    if [ ! -d sysconfcpus/bin ];
-    then
-      git clone https://github.com/obmarg/libsysconfcpus.git;
-      cd libsysconfcpus;
-      ./configure --prefix=$TRAVIS_BUILD_DIR/sysconfcpus;
-      make && make install;
-      cd ..;
-    fi
-  - cd aion
-  - mix local.rebar --force
-  - mix local.hex --force
-  - mix deps.get
-
-before_install:
+- |
+  if [ ! -d sysconfcpus/bin ];
+  then
+    git clone https://github.com/obmarg/libsysconfcpus.git;
+    cd libsysconfcpus;
+    ./configure --prefix=$TRAVIS_BUILD_DIR/sysconfcpus;
+    make && make install;
+    cd ..;
+  fi
 - cd aion
+- mix local.rebar --force
+- mix local.hex --force
+- mix deps.get
 - npm install
 
 before_script:
 - cp config/travis.exs config/test.exs
 - mix do ecto.create, ecto.migrate
 
 script:
-  - cd web/elm &&
-    ../../node_modules/elm/binwrappers/elm-package install --yes &&
-    $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 ../../node_modules/elm/binwrappers/elm-make --yes src/App.elm &&
-    rm -rf index.html &&
-    cd ../../
-  - mix credo --strict
-  - mix test
+- cd web/elm &&
+  ../../node_modules/elm/binwrappers/elm-package install --yes &&
+  $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 ../../node_modules/elm/binwrappers/elm-make --yes src/App.elm &&
+  rm -rf index.html &&
+  cd ../../
+- mix credo --strict
+- mix test
 
 notifications:
   email: false
   slack:
     secure: 0wrGLWdpp9HLujCNNwJnPvmfQmK3phQXbUj/EZ/x/YROEp1TdaDzLnXI4Qo6JyZszmH+zFb3S58iiOUrX2oCJFhdUGz/tN8YSBQBdMDPhi602BfarK4dfOSsnMq60KYzEV+zkM3GQFXIl/UudeZ5AhEpEn4tF6HrjSYqMOAJkjv4REesdY/GLDVSFfNF/KfXhd1afsB5IA5O0VMERr247aCtnr9pbHuE6yqicZCuw0aU41e3zPRm8BVUIP1np5jRaGCJs7Vpx/5Q+SpdTjqdtgB05APvHveupDJrGV2uVH2iXScxHIP5SZOJbaaUxkIvjjrMN4IW3IiFDjxDuN6PAFZfJ3mpTowtDl96t4fgOfTM9a/EVv7OlPeQAVkOvHIdfsZ2PUpGV7+yWN1RssyiaYxcHdJ825on2jb+lj7lhsZmlGDfa/8GhkPQuKJr/msaXaposYDa42460hinn4UPHGrgO4AuGrXF9Cc7/xbGIdAkHogsswYy1X2XDf7aDR5yktgz/sQ1aHnvF9K1SzopDd1TS6YjLpgN0Y4kqWrUqWxHM6Ne1mTYLTREfbs+49+oPaq+n+HilFB0BAjCpwtwvKi+VhPgH5ZCSYL0hqiYpsUNL2Q3oWUHra1YlPIc/2QtIf6E1XtpqADRqnI/Ow8XkTgi4xWq2gKmHn5jEZhW9Ng=
+
+deploy:
```

Comment:
Since we decided to go to different hosting service due to limitations on Heroku, I would delete this / save that for later as staging when we introduce Guardina.

Reply:
Agreed, will do.